main.go

```diff
@@ -171,7 +172,12 @@ func main() {
 		if err = cfg.VppInit.Decode("AF_PACKET"); err != nil {
 			log.FromContext(ctx).Fatalf("VppInit.Decode error: %v", err)
 		}
-		vppConn, vppErrCh = vpphelper.StartAndDialContext(ctx)
+		var tmpDir string
+		tmpDir, err = ioutil.TempDir("", cfg.Name)
+		if err != nil {
+			logrus.Fatalf("error creating tmpDir %+v", err)
+		}
+		vppConn, vppErrCh = vpphelper.StartAndDialContext(ctx, vpphelper.WithRootDir(tmpDir))
```

Comment:
Maybe we can just use here `vpphelper.WithRootDir(os.TempDir())`
because it's just a local folder